pages/__tests__/Profile.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Profile from '../profile'
+import '@testing-library/jest-dom'
+import { render, screen, waitFor } from '@testing-library/react'
+
+test('renders example component', async () => {
+  render(<Profile />)
+  const textElement = screen.getByText('profile')
+  expect(textElement).toBeInTheDocument()
+
+  await waitFor(() => {
+    expect(textElement).toBeInTheDocument()
+  })
+})
+
+test('Check if specific text appears on the page', () => {
+  const { getByText } = render(<Profile />)
+  const expectedText = 'profile'
+
+  expect(getByText(expectedText)).toBeInTheDocument()
```

Comment:
use `screen.getByText` for consistency and also:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen